CHANGELOG.md

```diff
@@ -2,6 +2,9 @@
 
 This file documents all notable changes in `LogDNA CloudWatch Lambda Function`. The release numbering uses [semantic versioning](http://semver.org).
 
+## v2.2.0 - Released on June 9, 2020
+* Add `RAW_EVENT_MESSAGE` environment variable option to set `line` to raw `event.message`
```

Comment:
Might be helpful to clarify what type this value can be. I'm guessing it's a boolean

Reply:
Oh I see, I guess it's [true,false,yes,no]